### PR TITLE
Update SPA code examples

### PR DESCRIPTION
**Pull request by ambarwootric**
Changes: Add more details in Single Page App (SPA) section `javascript#single-page-app-spa` of docs.

Trello: https://trello.com/c/Wwhp66lf/4821-check-for-outdated-docs-and-examples

### CODE REVIEW

File: source/includes/_install_js_spa_intro.md [hunk 1]
```diff
@@ -25,57 +25,167 @@
...
-<!-- begin Wootric code -->
-angular.module('wootricAngularApp')
-.factory('wootricFactory', function () {
-window.wootricSettings = {
-    email: 'nps@example.com',// TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
+// 1. Load Wootric JS SDK
+<script type="text/javascript" src="https://cdn.wootric.com/wootric-sdk.js"></script>
+
+// 2. Create Factory
+app.factory('wootricFactory', function() {
```
Comment (diegoserranoa) [inline source/includes/_install_js_spa_intro.md:35]: Where does the `app` var come from? I thought using `angular` was more general.

File: source/includes/_install_js_spa_intro.md [hunk 2]
```diff
@@ -25,57 +25,167 @@
...
+    email: 'customer@example.com',// TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
     created_at: 1234567890, // TODO: The current logged in user's sign-up date as a 10 digit Unix timestamp.
-    account_token: 'NPS-xxxxxxx' // This is your unique account token.
+    account_token: 'NPS-xxxxxxxx' // This is your unique account token.
   };
   return {
     run: function() {
-      window.wootric_survey_immediately = true; // TODO: Comment out for production.      
+      window.wootric_survey_immediately = true; // Shows survey immediately for testing purposes.  TODO: Comment out for production.
+      window.wootric_no_surveyed_cookie = true; // Bypass cookie based throttle for testing purposes.  TODO: Comment out for production.
```
Comment (diegoserranoa) [inline source/includes/_install_js_spa_intro.md:44]: Why are we including this in the example? Do customers need it regularly?

File: source/includes/_install_js_spa_intro.md (hunk 1 above)
Comment (ambarwootric) [reply source/includes/_install_js_spa_intro.md:35]: @diegoserranoa thanks for checking. These are code snippets and assumes that a developer looking at docs knows at least very basic also in point number for I have mentioned that there is code example for this on github and when `README` will be checked out & thus the code itself then there's `app` variable  https://github.com/Wootric/spa-examples/blob/master/angularjs/app/app.js#L4

File: source/includes/_install_js_spa_intro.md (hunk 2 above)
Comment (ambarwootric) [reply source/includes/_install_js_spa_intro.md:44]: It's more like that we have those to test out quickly and as `TODO` says to comment those out in production.

**Review by diegoserranoa — APPROVED** [review]
Thanks for answering my questions @ambarwootric !

**Pull request closed without merging**